enclave-runtime/src/error.rs

```diff
@@ -41,6 +41,7 @@ pub enum Error {
 	PrimitivesAccess(itp_primitives_cache::error::Error),
 	MutexAccess,
 	Other(Box<dyn std::error::Error>),
+	Attestation(itp_attestation_handler::error::Error),
```

Comment:
We usually have `Other` as last enum variant, so maybe move this new variant up.